lib/explain.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+/**
+ * @class
+ * @property {string} verbosity The verbosity mode for the explain output
+ */
+class Explain {
```

Comment:
Just a suggestion, we can bring over that enum from 4.0 and use Object.values() for example to check the string passed in for verbosity